modules/kafka/kafka.go

```diff
@@ -46,14 +63,34 @@ func RunContainer(ctx context.Context, opts ...testcontainers.ContainerCustomize
 
 // Run creates an instance of the Kafka container type
 func Run(ctx context.Context, img string, opts ...testcontainers.ContainerCustomizer) (*KafkaContainer, error) {
+	listeners := []string{
```

Comment:
suggestion: looks like these are only ever consumed as a comma separated string, so define them as that here do avoid multiple joins.

Reply:
I did it this way because I feel it makes it much much easier to read and confirm that the correct pairings of `${LISTENER_NAME} <> Port` are taking place compared to a big log string with multiple `fmt %` characters in it.

Reply:
To clarify I would keep them defined like you have, as you say its easier to read and maintain, but wrap with the `strings.Join` here so you done run that twice.